12/umbraco-forms/editor/viewing-and-exporting-entries.md

```diff
@@ -26,7 +26,7 @@ Clicking on the first field for each record in the list will open the full set o
 
 ### Editing the Entries
 
-If configured via the permissions model and supported by the version of Umbraco Forms you are running, entries may be editable via the back-office. If available, click the _Edit_ button to switch the read-only view of an entry to an editable one and _Save_ to record the changes. An audit trail will show who and when updates on the entry were made.
+If configured via the permissions model and supported by the version of Umbraco Forms you are running, entries may be editable via the backoffice. If available, click the _Edit_ button to switch the read-only view of an entry to an editable one and _Save_ to record the changes. An audit trail will show who and when updates on the entry were made.
```

Comment:
In order to keep the documentation consistent, can you please make this same change to version 10 and 11 of Forms as well? 😊

Reply:
Sure. I'll add the changes.